src/EnergyPlus/StandardRatings.cc

```diff
@@ -561,7 +564,13 @@ namespace StandardRatings {
 
         Power = (AvailChillerCap / Par(10)) * ReformEIRChillerEIRFPLR * ReformEIRChillerEIRFT;
 
-        QEvap = AvailChillerCap * Par(4);
+        if (Par(4) >= 1.0) {
+            PartLoadRatio = Par(4);
+        } else {
+            PartLoadRatio = Par(4) * Par(12) / ReformEIRChillerCapFT;
+        }
+
+        QEvap = AvailChillerCap * PartLoadRatio;
```

Comment:
Calculate the PLR inside the residual calculation function since the PLR depends on the CAP-f-T.

Reply:
Should the limit of Par(4)/PLR to 1 happen going into this function or tested on each iteration as done here? There will be at minimum 3 iterations. Then it reverts back to old line 721.

    Par(4) = std::min(1.0, ReducedPLR[RedCapNum]);

Reply:
This is block was not intended as a limit but just as a way to calculate the PLR within that residual calculation function (as opposed to before where it was calculated after the LCT was determined). `Par(4)` is the percent load per AHRI which is different from the PLR. The PLR depends on the CAP-f-T (which depends on the LCT) and is used to calculate `Qevap` so I think that it should be in this function. For rating purposes at 100% percent load per AHRI, we should be at a PLR of 1.

Reply:
I have been looking at the if (Par(4) block above. I just want to be sure that Standard Ratings reference capacity increases as it does in this branch.

Par(4) is passed in from an outer loop and is:

    Par(4) = ReducedPLR[RedCapNum]; which is a 4 element array of 1.0, 0.75, 0.5 and 0.25

In this function the Par used are:

    // par(4)  = Percent load per AHRI - so is either 0.25, 0.5, 0.75 or 1.0
    // par(6)  = Index for the total cooling capacity modifier curve
    // par(9)  = Reference capacity of chiller [W]
    // par(12) = CAP-f-T at reference/rated temperatures, these are what's called out in the standard as:

    // get the entering water temperature for the reduced capacity test conditions
    if (ReducedPLR[RedCapNum] > 0.50) {
        EnteringWaterTempReduced = 8.0 + 22.0 * ReducedPLR[RedCapNum];
    } else {
        EnteringWaterTempReduced = 19.0;
    }
    CondenserInletTemp = EnteringWaterTempReduced;
    ChillerCapFT_rated = CurveValue(state, CapFTempCurveIndex, EvapOutletTemp, CondenserInletTemp);

Then looking at this function the variables used in the QEvap equation are:

    ReformEIRChillerCapFT = CurveValue(state, int(Par(6)), EvapOutletTemp, CondenserOutletTemp);
    AvailChillerCap = Par(9) * ReformEIRChillerCapFT;
    PartLoadRatio = Par(4) * Par(12) / ReformEIRChillerCapFT;
    QEvap = AvailChillerCap * PartLoadRatio;

At Par(4) = 1 then using the if (Par(4) >= 1.0) then PartLoadRatio = Par(4) = 1:

    QEvap = AvailChillerCap * 1.0 or
    QEvap = RatedCapacity * ReformEIRChillerCapF
    this looks correct and capacity should increase because AvailChillerCap is based on the condenser outlet temp. OK.

Then for the else, if you expand QEvap terms you see that ReformEIRChillerCapFT drops out:

    QEvap = AvailChillerCap * PartLoadRatio;
    QEvap = Par(9) * ReformEIRChillerCapFT * Par(4) * Par(12) / ReformEIRChillerCapFT or
                = Par(9) * Par(4) * Par(12) or
                = RatedCapacity * Par(4) * CapFT,rated or
                = RatedCapacity * CapFT,rated * Par(4)

where (RatedCapacity * CapFT,rated) is based on the condenser inlet temp and does not account for the model's use of condenser outlet temp. OK, I see it now, CapFT,rated is the ReformEIRChillerCapFT from when Par(4) = 1 and that is based on the condenser outlet temp found on the first pass through.

![image](https://user-images.githubusercontent.com/5778156/190470962-e40ce282-dcb7-4ab0-9641-08e1eb8ebc21.png)

I took some numbers from the FourPipeBeamLargeOffice file and everything does look correct. And because Par(12) is 0 on the first pass through that % load needs to be treated separately. As least I can't think of a way around that.

![image](https://user-images.githubusercontent.com/5778156/190479996-63719d04-69e9-4165-9ec1-d3197eec3ba3.png)